salt_resources/states/conda/jupyter_server.sls

```diff
@@ -0,0 +1,56 @@
+{% set current_path = salt['environ.get']('PATH', '/bin:/usr/bin') %}
+{% set conda_path = '/home/' + grains['deescalated_user'] + '/miniconda3' %}
+{% set conda_bin_path = conda_path + '/bin' %}
+
+install_jupyter:
+  cmd.run:
+    - name: conda install -n conda_env -y jupyter -c anaconda
+    - runas: {{ grains['deescalated_user'] }}
+    - env:
+      - PATH: {{ [current_path, conda_bin_path]|join(':') }}
+    - require:
+      - sls: conda
+      - sls: conda.anaconda
+
+create_jupyter_root_dir:
+  file.directory:
+    - name: /home/{{ grains['deescalated_user'] }}/.jupyter
+    - user: {{ grains['deescalated_user'] }}
+    - group: {{ grains['deescalated_user'] }}
+
+create_jupyter_runtime_dir:
+  file.directory:
+    - name: /var/tmp/jupyter_runtime
+    - user: {{ grains['deescalated_user'] }}
+    - group: {{ grains['deescalated_user'] }}
+
+place_jupyter_notebook_config_py_file:
+  file.managed:
+    - name: /home/{{ grains['deescalated_user'] }}/.jupyter/jupyter_notebook_config.py
+    - source: salt://conda/jupyter_notebook_config.py
```

Comment:
The file it points to is commented out, not this line literally. Expand the diff right above this in the pull request.

Reply:
right, the whole file is a stock template that defaults ton be completely commented out. jupyter needs a python file in this location with this name. It errors with out the file.

we could make the file be empty, but then it would not have any nice optional configs in it that are ready to go.